diffusers/training_example.ipynb

```diff
@@ -0,0 +1,3955 @@
+{
```

Comment:
<p>Can trim up the text about the UNet time encoding to make it clearer. Maybe add a figure if possible.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/notebooks/pull/205/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
Updated...